Rebuild-Black-Business/RBB-Website
# Fix/biz ally citystate

# Describe your PR

Removes zipcode package, adds newly passed City / State values to Business and Allies cards


## Pages/Interfaces that will change

/allies
/businesses

## Steps to test

1. Navigate to /allies, ensure City / State string shows up on cards
2. Navigate to /businesses, ensure City / State string shows up on cards

### Additional notes
